content/version_control/VCS.tex

```diff
@@ -0,0 +1,174 @@
+
+\documentclass{beamer}
+
+\usetheme{default}
+
+\title{Version Control Systems(VCS)}
+
+\author{Kiran Vasudev\inst{1}}
+
+\institute[Universities of Somewhere and Elsewhere] 
+{
+  \inst{1}
+  Hochschule Bonn-Rhein-Sieg
+
+}
+
+\begin{document}
+
+\begin{frame}
+  \titlepage
+\end{frame}
+
+\begin{frame}{Outline}
+  \tableofcontents
+\end{frame}
+
+\begin{frame}{What is it?}
+\section{What is it?}
+  \begin{itemize}
+  \item {
+    A method to manage file changes over time. These files are stored in a central location.
+  }
+  \item {
+    A backup when things go wrong.
+  }
+  \item{A software that helps team members collaborate with minimum disruptions.}
+  \end{itemize}
+\end{frame}
+
+\begin{frame}{Why should we use it ?}
+\section{Why should we use it ?}
+  \begin{itemize}
+  \item {
+    A complete long-term change history of every file.
+  }
+  \item {   
+    Branching and merging
+  }
+  \item {   
+    Ability to trace each change made
+  }
+  \item {   
+	Allows multiple developers to work simultaneously
+  }
+  \end{itemize}
+\end{frame}
+
+\begin{frame}{Types of VCS}{Centralized VCS}
+\section{Types of VCS}
+\subsection{Centralized VCS}
+  \begin{columns}
+\column{0.5\textwidth}
+\begin{itemize}
+  \item {
+    Centralized VCS keeps the history of changes on a central server.
+  }
+  \item {
+    Designed with the intent that there is only one true source  
+  }
+  \item {
+    If you want to make a copy of your data, you have to copy/paste it, literally.  
+  }
+  \item{
+	Sourceforge.net uses this type of versioning.  
+  }
+  \item{
+      \textbf{Example:} SVN (Subversion)
+  }  
+  \end{itemize}
+\column{0.5\textwidth}
+\begin{figure}
+	 \includegraphics[width=.9\textwidth]{images/cvcs}
+	 \caption{Centralized Version System \cite{website}}
+ \end{figure}
+\end{columns}
+\end{frame}	
+
+\begin{frame}{Types of VCS}{Decentralized VCS}
+\subsection{Distributed VCS}
+\begin{columns}
+\column{0.5\textwidth}
+\begin{itemize}
+		  \item {
+		    In distributed VCS, everyone has a local copy of the entire work’s history
+		  }
+		  \item {
+		    Each repository is as good as the other
+		  }
+		  \item{
+			Prone to central server crash
+		  }
+		  \item{
+			Your local copy is a repository, and you can commit to it and get all benefits of source control - Offline Source Control.
+		  }	  
+		  \item{
+			Mozilla Firefox uses this kind of versioning.  
```

Comment:
I think the message will be clearer if you say "Millions of projects on GitHub use this kind of versioning" rather than just giving a single example.